js_modules/dagster-ui/packages/ui-components/src/components/HorizontalControls.tsx

```diff
@@ -0,0 +1,18 @@
+import {Children, ReactNode} from 'react';
+
+import {Box} from './Box';
+import styles from './css/HorizontalControls.module.css';
+
+interface Props {
+  children: ReactNode;
+}
+
+export const HorizontalControls = ({children}: Props) => {
+  return (
+    <Box flex={{direction: 'row', gap: 8, alignItems: 'center'}} className={styles.container}>
+      {Children.map(children, (child) =>
```

Comment:
hmm, not a big fan of using children this way, would rather just have a prop of ReactNode[] | ReactNode I guess?

Reply:
Yeah, I'm fine with that too.